src/main/java/com/desiato/puresynth/controllers/LoginController.java

```diff
@@ -20,7 +22,10 @@ public class LoginController {
 
     @PostMapping
     public ResponseEntity<LoginResponseDTO> authenticateUser(@RequestBody AuthenticationRequestDTO requestDTO) {
+        log.info("Received login request for email: {}", requestDTO.email());
         PureSynthToken pureSynthToken = authenticationService.authenticate(requestDTO);
+        log.info("Authenticate method triggered");
```

Comment:
What is the value of this log?

Reply:
all the logging statements where added to understand exactly the flow of the program. I added them before starting the debugging to see if they were triggered through the process

Reply:
In a real application, when you have 10s or 100s requests per second, such log without any ids are usually rarely useful. Also to understand application flow, it's much better to use debugger than logging.